src/index.html

```diff
@@ -7,6 +7,308 @@
     <link rel="stylesheet" href="./styles/main.scss">
```

Comment:
SASS/SCSS is a CSS preprocessor that runs on the server and compiles to CSS code that your browser understands. Usually, the <link> tag should reference a compiled CSS file rather than the SCSS source file. Make sure to compile your SCSS file into CSS before linking it in your HTML.